packages/drivers/odsp-driver/src/test/createNewUtilsTests.spec.ts

```diff
@@ -96,7 +96,8 @@ describe("Create New Utils Tests", () => {
                 docId: hashedDocumentId,
                 resolvedUrl,
             },
-            new TelemetryNullLogger());
+            new TelemetryNullLogger(),
+            undefined);
```

Comment:
You added tests for the snapshot expiry right? You can probably pass in the timeout via the policy in those now.
Basically, we need tests to validate that the policy works. Also, a test to validate that a value higher than default results in an error.